File: C/coordenadas.c
#include <stdio.h>
#include <locale.h>

double x, y;

int main() {
    setlocale(LC_ALL, "Portuguese");

    printf("Digite o valor de X: ");
    scanf("%lf", &x);
    printf("Digite o valor de Y: ");
    scanf("%lf", &y);

    if (x == 0 && y == 0) {
        printf("Origem\n");
    } else if (x == 0) {
        printf("Eixo Y\n");
    } else if (y == 0) {
        printf("Eixo X\n");
    } else if (x > 0 && y > 0) {
        printf("Q1\n");
    } else if (x < 0 && y > 0) {
        printf("Q2\n");
    } else if (x < 0 && y < 0) {
        printf("Q3\n");
    } else {
        printf("Q4\n");
    }

}

File: Python/coordenadas.py
x = float(input("Digite o valor de x: "))
y = float(input("Digite o valor de y: "))

if x == 0 or y == 0:
    print("Origem\n")
elif x == 0:
    print("Eixo Y")
elif y == 0:
    print("Eixo X")
elif x > 0 and y > 0:
    print("Q1\n")
elif x < 0 < y:
    print("Q2\n")
elif x < 0 and y < 0:
    print("Q3\n")
else:
    print("Q4\n")

